index.html

```diff
@@ -479,9 +479,11 @@ <h2>This is an interactive API</h2>
                   let p = curve.get(t);
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
+                  console.log(Math.abs(kr.k))
+                  if (Math.abs(kr.k) <= 0.015) continue;
```

Comment:
this optimization is not necessary for demonstrator code.

Reply:
Remove it